shell-script/useless-vm-check.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# ENVIRONMENT VARIABLES:
+# 
+#   * AWS_ACCESS_KEY, required, the AWS access key
+#   * AWS_SECRET_KEY, required, the AWS secret key
+#   * GITHUB_TOKEN, required, GitHub API authentication token
+set -euo pipefail
+AWS_ACCESS_KEY=${AWS_ACCESS_KEY_ID}
+AWS_SECRET_KEY=${AWS_SECRET_ACCESS_KEY}
+GITHUB_TOKEN=${GITHUB_TOKEN}
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+function markdown_report() {
+    case "$OSTYPE" in
+        darwin*)
+            if ! type gsed &> /dev/null || ! type ggrep &> /dev/null; then
+                echo "GNU sed and grep not found! You can install via Homebrew" >&2
+                echo >&2
+                echo "    brew install grep gnu-sed" >&2
+                exit 1
+            fi
+
+            SED=gsed
+            GREP=ggrep
+            ;;
+        *)
+            SED=sed
+            GREP=grep
+            ;;
+    esac
+    echo "**Daily VM check List**:"
+    echo "| Tag | Instance_id | state |"
+    echo "| :--- | :--- | :--- |"
+    cat ${GITHUB_WORKSPACE}/instances.txt
+}
+
+# Upload report through GitHub issue comment
+function github_add_comment() {
+    report="$1"
+    export GITHUB_TOKEN=${GITHUB_TOKEN}
+    $SCRIPT_PATH/ok.sh add_comment nervosnetwork/ckb-internal 1665 "$report"
+}
+function main() {
+  aws configure set aws_access_key_id ${AWS_ACCESS_KEY}
+  aws configure set aws_secret_access_key ${AWS_SECRET_KEY}
+  aws configure set region us-west-2
+  aws ec2 describe-regions --output text --query Regions[*].[RegionName] >>regions.txt
+  while read -r region;
+  do
+    region=$(echo "$region" | tr -d '[:space:]')
+    aws ec2 describe-instances --region $region |jq '.Reservations[].Instances[] | "| \(.Tags[].Value) | \(.InstanceId) | \(.State.Name) |"'| sed 's/\"//g' >>${GITHUB_WORKSPACE}/instances.txt
```

Comment:
Can it also list other kinds of resources like VPC, keypair, sg?

Reply:
sure，will have a try.